TicTacToe/src/tictactoe/TicTacToe.java

```diff
@@ -4,7 +4,10 @@
     
     private char[][] board;
 
-    // Constructors
+    /**
```

Comment:
Missing documentation at top of class and documentation of board member variable.